cclib/parser/orcaparser.py

```diff
@@ -793,7 +794,11 @@ def splitter(line):
                 line = next(inputfile)
             thermal_enthalpy_correction = float(next(inputfile).split()[4])
             next(inputfile)
-            self.enthalpy = float(next(inputfile).split()[3])
+
+            if self.natom > 1:
+                self.enthalpy = float(next(inputfile).split()[3])
+            else:
+                self.enthalpy = self.electronic_energy + thermal_translational_correction
```

Comment:
Enthalpy doesn't seem to be affected (at least in [the regression file](https://github.com/cclib/cclib-data/blob/master/ORCA/ORCA4.1/single_atom_freq.out#L879)). So do we need this?

Reply:
Removed in #818.

Reply:
This looks innocent but is in fact a dangerous value. It includes the [total free energy (i.e. inner energy)](https://github.com/cclib/cclib-data/blob/25cb852e143e4e2e790aa415b1b0e0ca6c190603/ORCA/ORCA4.1/single_atom_freq.out#L876), which includes the [spurious vibrational correction](https://github.com/cclib/cclib-data/blob/25cb852e143e4e2e790aa415b1b0e0ca6c190603/ORCA/ORCA4.1/single_atom_freq.out#L855) that ORCA thought was nonzero (because [it could not remove the translational modes](https://github.com/cclib/cclib-data/blob/25cb852e143e4e2e790aa415b1b0e0ca6c190603/ORCA/ORCA4.1/single_atom_freq.out#L773-L777)).

This means that the enthalpy ORCA reports is wrong by 1.78 kcal/mol. It even calculates a ZPE which, although almost zero in this regression file, might not be in others (I'm not sure, depends on the spurious translational frequencies, which probably depend on atomic masses, I would have to look closer).

In the end, the [only correction to electronic energy that is valid is the translational one](https://github.com/cclib/cclib-data/blob/25cb852e143e4e2e790aa415b1b0e0ca6c190603/ORCA/ORCA4.1/single_atom_freq.out#L854-L857), which what I included.